### PR TITLE
Polish microarray GSVA example

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
### Purpose

Polishing the microarray GSVA example.

### Issue addressed

For the microarray GSVA example, the following are being addressed:

- Enforcing comment character length limit (#406)
- Stopping row names from being printed for paged tables (#428)
- Adding the introductory paragraphs that are in every pathway analysis notebook (#371)

### Comments/questions for the reviewer

Outside of copying over some of the changes to the microarray differential expression example with more than 2 groups (#427), I have not done much with the `limma` section.

### CODE REVIEW

File: 02-microarray/pathway-analysis_microarray_03_gsva.Rmd [hunk 1]
```diff
@@ -11,19 +11,34 @@ output:
...
+In the context of [refine.bio](https://www.refine.bio/), we use these techniques to analyze and interpret genome-wide gene expression experiments.
+The rationale for performing pathway analysis is that looking at the pathway-level may be more biologically meaningful than considering individual genes, especially if a large number of genes are differentially expressed between conditions of interest.
+In addition, many relatively small changes in the expression values of genes in the same pathway could lead to a phenotypic outcome and these small changes may go undetected in differential gene expression analysis.
+
+We highly recommend taking a look at [Ten Years of Pathway Analysis: Current Approaches and Outstanding Challenges](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1002375) from @Khatri2012 for a more comprehensive overview. We have provided primary publications and documentation of the methods we will introduce below as well as some recommended reading in the [`Resources for further learning` section](#resources-for-further-learning).
```
Comment (cansavvy) [inline 02-microarray/pathway-analysis_microarray_03_gsva.Rmd:29]:
```suggestion
We highly recommend taking a look at [Ten Years of Pathway Analysis: Current Approaches and Outstanding Challenges](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1002375) from @Khatri2012 for a more comprehensive overview. 
We have provided primary publications and documentation of the methods we will introduce below as well as some recommended reading in the [`Resources for further learning` section](#resources-for-further-learning).
```

Comment (cansavvy) [reply 02-microarray/pathway-analysis_microarray_03_gsva.Rmd:29]: New line is all. (Perhaps this is like this elsewhere though?)

**Review by cansavvy — APPROVED** [review]
Just minor things. I like the de-messy-ing of the design matrix column names. Good call.

File: 02-microarray/pathway-analysis_microarray_03_gsva.Rmd (hunk 1 above)
Comment (jaclyn-taroni) [reply 02-microarray/pathway-analysis_microarray_03_gsva.Rmd:29]: I assume that this is like this in all of the pathway notebooks then. But I can check. It's almost certainly like this in the ones I put in where I copied and pasted it from _somewhere_.

**Comment by jaclyn-taroni** [comment]
> I like the de-messy-ing of the design matrix column names. Good call.

Can't take credit for this, all @jashapiro on #427!

File: 02-microarray/pathway-analysis_microarray_03_gsva.Rmd [hunk 2]
```diff
@@ -121,6 +144,7 @@ Your new analysis folder should contain:
     - The metadata TSV  
 - A folder for `plots` (currently empty)  
 - A folder for `results` (currently empty)  
+- A folder for `genesets` (currently empty)
```
[the remark this answers is not in the document]
Comment (jaclyn-taroni) [reply 02-microarray/pathway-analysis_microarray_03_gsva.Rmd:147]: Good catch, thanks!

**Pull request merged**